Sources/MongoSwift/MongoCollection.swift

```diff
@@ -133,4 +133,92 @@ public class MongoCollection<T: Codable> {
 
         return try body(collection)
     }
+
+    /**
+     * Starts a `ChangeStream` on a collection. By default, the type `CollectionType` is associated with the
+     * `fullDocument`field in `ChangeStreamsDocument`.
+     * - Parameters:
+     *   - Pipeline: The pipeline of stages to append to an initial `ChangeStream` stage.
+     *   - Options: An optional `ChangeStreamOptions` to use on the initial `ChangeStream` stage.
+     *   - Session: An optional `ChangeStream` to use with this change stream.
+     * - Returns: A change stream on a specific collection.
+     * - Throws:
+     *   - `ServerError.commandError` if an error occurs on the server while creating the change stream.
```

Comment:
Since a `ServerError` can be thrown here, that implies that we're doing i/o as part of this function call. Generally we like to encapsulate all our i/o in `Operation`s, so we should probably do that here, too. 

If we're not performing I/O, then this error case should be impossible.

Reply:
`watch` does indeed send the initial aggregate command to the server so ya I/O is happening

I think the design of an appropriate operation type that can be used for all the commands is non-trivial (should there be separate ones for client, db, coll? should we combine them in a single op and use an enum?) so I'd be ok with adding a follow on ticket for refactoring the watch calls into an operation rather than doing it right now

Reply:
Yeah, unless I overlooked, it seems like `mongoc_watch` does not call the server, only creates the change stream cursor and prepares the commands to be sent later?
I have removed the `ServerError` from the docstrings for the `watch` methods.

Reply:
Leaving it for a follow on ticket sounds good to me. As part of that we'll also need a `ChangeStreamNextOperation` (or modification to the existing `NextOperation`) too.

Reply:
Just created two tix. One for the `watch` operation ([SWIFT-521](https://jira.mongodb.org/browse/SWIFT-521)), and one for `ChangeStreamNextOperation` ([SWIFT-523](https://jira.mongodb.org/browse/SWIFT-523)).